coalib/collecting/Collectors.py

```diff
@@ -142,19 +139,17 @@ def icollect_bears(bear_dir_glob, bear_globs, kinds, log_printer):
                          'the latter solution might break other python '
                          'packages that depend on the currently installed '
                          'version.').format(file=matching_file,
-                                            pkg=exception.req),
-                        exception, log_level=LOG_LEVEL.WARNING)
+                                            pkg=exception.req))
+                    LogHelper.log_exception(exception)
```

Comment:
wouldn't it be simpler if log_exception did both the warning and the info message? We want that pattern consistently and if we decide to change that it's easier to do that all in one place.

Second, log_exception could just take the last thrown exception by default with an optional exception argument that we can give to override.